src/main/java/com/northconcepts/datapipeline/foundations/examples/datamapping/CaptureDataThatFailedDataMappingValidation.java

```diff
@@ -6,45 +6,53 @@
  */
 package com.northconcepts.datapipeline.foundations.examples.datamapping;
 
-import com.northconcepts.datapipeline.core.DataReader;
+import java.math.BigDecimal;
+
 import com.northconcepts.datapipeline.core.FieldType;
-import com.northconcepts.datapipeline.core.NullWriter;
 import com.northconcepts.datapipeline.core.StreamWriter;
-import com.northconcepts.datapipeline.csv.CSVReader;
 import com.northconcepts.datapipeline.foundations.datamapping.DataMapping;
-import com.northconcepts.datapipeline.foundations.datamapping.DataMappingReader;
 import com.northconcepts.datapipeline.foundations.datamapping.FieldMapping;
+import com.northconcepts.datapipeline.foundations.file.LocalFileSink;
+import com.northconcepts.datapipeline.foundations.file.LocalFileSource;
+import com.northconcepts.datapipeline.foundations.pipeline.DataMappingPipeline;
+import com.northconcepts.datapipeline.foundations.pipeline.input.CsvPipelineInput;
+import com.northconcepts.datapipeline.foundations.pipeline.output.ExcelPipelineOutput;
 import com.northconcepts.datapipeline.foundations.schema.EntityDef;
 import com.northconcepts.datapipeline.foundations.schema.NumericFieldDef;
 import com.northconcepts.datapipeline.job.Job;
 import com.northconcepts.datapipeline.memory.MemoryReader;
 import com.northconcepts.datapipeline.memory.MemoryWriter;
 
-import java.io.File;
-import java.math.BigDecimal;
-
 public class CaptureDataThatFailedDataMappingValidation {
```

Comment:
@DeleTaylor Do I have to move this example and next example to pipeline package?